controllers/configController.ts

```diff
@@ -11,11 +24,12 @@ function getConfig(req: Request, res: Response): void {
     dateOfBirthDeploymentDate: config.dateOfBirthDeploymentDate,
     sessionDuration: config.sessionDurationHours,
     satistics: config.satistics,
+    universities,
```

Comment:
Why add it in config and not call it in ConventionForm the only place where it is used?

Reply:
because i was too lazy to make a new endpoint :(